constants/constants.go

```diff
@@ -7,8 +7,11 @@ const (
 	AUTOMATE_PRODUCT      = "automate"
 	HABITAT_PRODUCT       = "habitat"
 	LATEST                = "latest"
+	PLATFORM_SERVICE      = "chef-360"
```

Comment:
There is a suffix `_PRODUCT` for all other products. So can we use `PLATFORM_SERVICE_PRODUCT` instead of just `PLATFORM_SERVICE`?